docs/sidebars.js

```diff
@@ -47,6 +47,14 @@ module.exports = {
         'ui/control-center/connectors',
       ],
     },
+    {
+      '🤖 Conversational AI connectors': [
+        'conversational-ai-connectors/overview',
+        'conversational-ai-connectors/cognigy-ai',
+        'conversational-ai-connectors/ibm-watson-assistant',
+        'conversational-ai-connectors/rasa',
+      ],
+    },
```

Comment:
Small suggestion, as I think `Conversational AI connectors` is a bit long for the side menu, we can perhaps add on top level `Connectors`, then under that we can just move the `Sources` and place `Conversational AI` on the same level as Sources (under Connectors).

Reply:
The `Sources`->`Connectors` page can go under the top level `Connectors` for now.

Reply:
great idea 👍 I'll make the fix!